lib/core/filtering/simple_rules/validation/no_conflicting_policies_rules_validator.rb

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+# frozen_string_literal: true
+
+require 'core/filtering/simple_rules/validation/simple_rules_validator'
+require 'core/filtering/validation_status'
+
+module Core
+  module Filtering
+    module SimpleRules
+      module Validation
+        class NoConflictingPoliciesRulesValidator < Core::Filtering::SimpleRules::Validation::SimpleRulesValidator
+
+          def are_rules_valid
+            rule_field_value_to_policy = {}
+
+            @rules.each do |simple_rule|
+              rule_field_value = simple_rule.slice('rule', 'field', 'value')
+              policy = simple_rule['policy']
+
+              return conflicting_rules(rule_field_value) if rule_field_value_to_policy.key?(rule_field_value) && rule_field_value_to_policy[rule_field_value]['policy'] != policy
+
+              rule_field_value_to_policy[rule_field_value] = policy
+            end
+
+            { :state => Core::Filtering::ValidationStatus::VALID, :errors => [] }
+          end
+
+          private
+
+          def conflicting_rules(rule_type_value)
+            {
+              :state => Core::Filtering::ValidationStatus::INVALID,
+              :errors => [
+                :ids => [SIMPLE_RULES_ID],
```

Comment:
This should actually be the IDs of the two rules that conflicted. Like if you had:
```
{
  {
    "id": "foo",
    "policy":"include",
    "field":"bar",
    "rule":"equals",
    "value":"baz",
    "order":1
  },
  {
    "id": "qux",
    "policy":"exclude",
    "field":"bar",
    "rule":"equals",
    "value":"baz",
    "order":2
  }
]
```

you should get `:ids => ['foo', 'qux']`

Reply:
@sphilipse I believe you'd mentioned that Kibana cannot handle these errors in 8.6, so changing this behavior is probably not a high priority right now. But maybe worth filing an issue to come back to this in early in 8.7.

Reply:
Good point, will create an issue for that